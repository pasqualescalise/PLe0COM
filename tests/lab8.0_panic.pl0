BEGIN
	if 1 > 2 then begin
		print "Okay";
	end else begin
		panic(34, "Panic: not expecting 1 <= 2");
	end;

	print "This should not print";
END
