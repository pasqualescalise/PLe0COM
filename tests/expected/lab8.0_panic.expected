Panic: not expecting 1 <= 2
